### ISSUE TITLE
Better direction?

### ISSUE THREAD

**Issue by tzwel** [grantcarthew/awesome-unique-id#14]
Thinking about submitting to [Awesome](https://github.com/sindresorhus/awesome) but that would require substantial changes, such as renaming the master branch, and also a better direction for the project.

Currently the name of the list is `awesome-unique-id ` which seems language-agnostic, however almost every submission is for JavaScript. What about renaming the list to something like `awesome-unique-id-js`, `awesome-js-unique-id` or `awesome-javascript-id` (my personal favorite)?

**Comment by grantcarthew** [comment]
I made this list when I was working with JS heavily. I had it as a private file on my computer initially. Once the whole Awesome thing popped up, I created this repo, but to be an official Awesome repo there were a couple of requirements I didn't quite meet. I lost interest as I moved on to other things.

The RoboHash site is just fantastic and I love it. I use it now and again. My current work for example has a CI/CD automation user who didn't have a profile image. It's great for that sort of thing.

I planned to add more ID technologies, but life got in the way. For example, my current struggle is getting screenshots working on Wayland. It is such a dogs breakfast that I'm going to put that off until support has improved I think. I digress.

Is there an awesome id project written in Golang? What about Rust? I would say there are a heap of other tech id packages we could add.

Some of those research papers you linked to could be added also.

I don't think I want to turn this into JAJSR (just another js repo). Id technology and all that surrounds it is fascinating.

Let's keep it generic. In fact, please add research papers as a heading @tzwel.

I will rename the master branch right now.

**Comment by grantcarthew** [comment]
The master branch has been renamed to main. Please update your local clones.

**Comment by tzwel** [comment]
Got it, added in [ffa9e05](https://github.com/grantcarthew/awesome-unique-id/commit/ffa9e051c9e6d39938e64744cdc5279c760865fa)

I'll keep this open in case we want to turn this completely Awesome-friendly and submit in the future. Maybe then people will contribute libs for other languages

**Comment by tzwel** [comment]
So to be sure, it's fine if I make changes to comply with the [Awesome list guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md)?

**Comment by grantcarthew** [comment]
Yes, make it your own. If you are keen on making it a JS only repo, let me know and I'll rename it. I'm not too invested in this. Happy to have someone else run with it and improve what they can. Thanks for you help.

**Comment by tzwel** [comment]
Let's stick with your idea and not make this js-only, it might be beneficial in the future.

> Thanks for you help.

You are welcome, it's very fun!

**Comment by tzwel** [comment]
```
  README.md:1:84
  ×   1:1   License was not detected by GitHub  remark-lint:awesome-github
  ×   1:1   Forbidden license section found     remark-lint:awesome-license
  ×   1:84  #contributors                       remark-lint:double-link
  ×  13:3   #contributors                       remark-lint:double-link
  ```
Here's what we have to fix to adhere to awesome list guidelines. Relicensing mandatory, and you choose what to do with the contributors sections

**Comment by grantcarthew** [comment]
I'm not running the linter tzwel, but that should do it.

**Comment by tzwel** [comment]
All good now!
